components/automate-ui/src/styles.scss

```diff
@@ -196,3 +196,68 @@ chef-loading-spinner[fixed] {
 input {
   line-height: normal;
 }
+
+// Angular Material Overrides
+.chef-control-menu-x {
+  position: absolute;
+  margin-top: 35px;
+  right: -48px;
+  margin-right: 35px;
+}
+
+[panelclass="chef-control-menu-x"] {
+
+  .mat-select-trigger {
+    display: inline-block;
+    border-radius: var(--chef-border-radius);
+    background-color: var(--chef-white);
+    width: 32px;
+    height: 32px;
+    cursor: pointer;
+    outline: none;
+
+    &:after {
+      content: '...';
+      position: absolute;
+      color: black;
+      bottom: 33%;
+      left: 30%;
+    }
+
+    &:hover {
+      background-color: #DEE5FD;
```

Comment:
@susanev several of these colors here are ones that I haven't seen used in the codebase yet.  Are there names you want to use for them when I make the variables?